river/metrics/preq_roc_auc.py

```diff
@@ -0,0 +1,67 @@
+from river import metrics, utils
+
+from .efficient_preqrocauc import EfficientPreqROCAUC
+
+__all__ = ["PreqROCAUC"]
+
+
+class PreqROCAUC(metrics.base.BinaryMetric):
+    """Prequential version of the Receiving Operating Characteristic Area Under the Curve.
+
+    The Prequential ROCAUC (predictive sequential) calculates the metric using the instances
+    in its window of size S. It keeps a queue of the instances, when a instance is added
+    and the queue length is equal or bigger than S, the last instance is removed. The metric
+    has a tree with ordered instances, in order to calculate the metric efficiently. It was
+    implemented based on the algorithm presented in Brzezinski and Stefanowski, 2017.
+
```

Comment:
Could you add a little paragraph to explain that this is an exact implementation, as opposed to the existing `ROCAUC` which is approximate? Users need to understand this trade-off.

Reply:
Added a paragraph explaining the difference to the standard ROCAUC.